lattices/Cargo.toml

```diff
@@ -15,3 +15,4 @@ serde = ["dep:serde"]
 cc-traits = "2.0.0"
 sealed = "0.5"
 serde = { version = "1.0.160", features = ["derive"], optional = true }
+itertools = "*"
```

Comment:
Will need to be set to a value for publishing, "0.10" is what the other crates are using, although maybe we should update to "0.12"

Reply:
Actually it doesn't seem like itertools is actually being used right now?